packages/meteor-wallet/src/lib/meteor-wallet.ts

```diff
@@ -62,15 +62,14 @@ const createMeteorWalletInjected: WalletBehaviourFactory<
       return [];
     }
 
```

Comment:
When tested It looked like there no issues with meteor-wallet, so maybe it shouldn't be updated. Can you confirm that and revert the change if that's the case?